### PR TITLE
Azure OpenAI v2 pipeline component update

### PR DESCRIPTION
**Pull request by PavanReddy28**
1. Exposing `learning_rate_multiplier` and `batch_size` parameters in oai v2 finetune pipeline component
2. Upgrading `openai_completions_finetune` component version from 0.3.5 to 0.3.10
3. Renamed `epochs` parameter to `n_epochs` to align with OpenAI

### CODE REVIEW

**Review by anagarajan97 — DISMISSED** [review]
changes look fine

**Comment by skasturi** [comment]
Don't you need to change the pipeline component version as well?

**Comment by skasturi** [comment]
> version: 0.0.8

Update version also? 

---
Refers to: assets/large_language_models/components_pipelines/oai_v2_1p/openai_completions_finetune_pipeline/spec.yaml:3 in ac6d92b. [](commit_id = ac6d92b03951b2ce291caa5121cf69ddb99675ce, deletion_comment = False)

File: assets/large_language_models/components_pipelines/oai_v2_1p/openai_completions_finetune_pipeline/spec.yaml [hunk 1]
```diff
@@ -63,13 +73,15 @@ jobs:
...
     type: command
-    component: azureml://registries/azure-openai-v2/components/openai_completions_finetune/versions/0.3.5
+    component: azureml://registries/azure-openai-v2/components/openai_completions_finetune/versions/0.3.10
     inputs:
       input_dataset: ${{parent.jobs.openai_data_import.outputs.out_dataset}}
       model: ${{parent.inputs.model}}
       task_type: ${{parent.inputs.task_type}}
       registered_model_name: ${{parent.inputs.registered_model_name}}
-      epochs: ${{parent.inputs.epochs}}
+      n_epochs: ${{parent.inputs.n_epochs}}
```
Comment (skasturi) [inline assets/large_language_models/components_pipelines/oai_v2_1p/openai_completions_finetune_pipeline/spec.yaml:82]: Is this not a breaking change?

File: assets/large_language_models/components_pipelines/oai_v2_1p/openai_completions_finetune_pipeline/spec.yaml [hunk 2]
```diff
@@ -37,11 +37,21 @@ inputs:
     type: string
     optional: False
     description: User-defined registered model name
-  epochs:
+  n_epochs:
     type: integer
     optional: False
```
Comment (skasturi) [inline assets/large_language_models/components_pipelines/oai_v2_1p/openai_completions_finetune_pipeline/spec.yaml:42]: >optional: False [](http://example.com/codeflow?start=4&length=15)

Given that there is a default value, isn't this Optional?

**Review by skasturi — CHANGES_REQUESTED** [review]
:clock1:

File: assets/large_language_models/components_pipelines/oai_v2_1p/openai_completions_finetune_pipeline/spec.yaml (hunk 1 above)
Comment (anagarajan97) [reply assets/large_language_models/components_pipelines/oai_v2_1p/openai_completions_finetune_pipeline/spec.yaml:82]: Yes. We will follow up with the corresponding team for making the appropriate changes that are needed as a result of this. Currently there are no customers for this pipeline component so there is no impact.

**Review by skasturi — APPROVED** [review]
:shipit:

**Pull request merged**